src/api/app/assets/stylesheets/webui/responsive_ux/sponsors.scss

```diff
@@ -0,0 +1,40 @@
+#sponsor {
+    background: #fff;
+}
+
+.sponsors {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
```

Comment:
Because I'm still thinking in non-bootstrap fashion. I'll fix it.

Reply:
I can only use bootstrap classes for one of those, because the rest of them are tied to the breakpoint rules in the css, and if I "hardcode" them in the layout, then I lose the breakpoint behaviour.

Reply:
But, I might try to `@extend` some of bootstrap's classes here...